images/ceph/toolbox.sh

```diff
@@ -46,19 +41,25 @@ EOF
 
 # watch the endpoints config file and update if the mon endpoints ever change
 watch_endpoints() {
-    # get the timestamp for the target of the soft link
+  # get the timestamp for the target of the soft link
+  real_path=$(realpath ${MON_CONFIG})
+  initial_time=$(stat -c %Z "${real_path}")
+  while true; do
     real_path=$(realpath ${MON_CONFIG})
-    initial_time=$(stat -c %Z ${real_path})
-    while true; do
-       real_path=$(realpath ${MON_CONFIG})
-       latest_time=$(stat -c %Z ${real_path})
-
-       if [[ "${latest_time}" != "${initial_time}" ]]; then
-         write_endpoints
-         initial_time=${latest_time}
-       fi
-       sleep 10
-    done
+    latest_time=$(stat -c %Z "${real_path}")
+    
+    if [[ "${latest_time}" != "${initial_time}" ]]; then
+      write_endpoints
+      initial_time=${latest_time}
+    fi
+    
+    # Enable or disable Ceph cluster debug logs
+    if [ -n "$CEPH_DEBUG_LEVEL" ] && [ ! -f /etc/ceph/set_ceph_debug_done ]; then
```

Comment:
Maybe a comment here also to mention the `set_ceph_debug_level.sh` script creates the `set_ceph_debug_done` file, just to remind future devs that the relationship exists.